redfish/computersystem.go

```diff
@@ -568,7 +568,6 @@ func (computersystem *ComputerSystem) UnmarshalJSON(b []byte) error {
 	computersystem.resetTarget = t.Actions.ComputerSystemReset.Target
 	computersystem.SupportedResetTypes = t.Actions.ComputerSystemReset.AllowedResetTypes
 	computersystem.setDefaultBootOrderTarget = t.Actions.SetDefaultBootOrder.Target
-	computersystem.ManagedBy = t.Links.ManagedBy.ToStrings()
```

Comment:
This just removes ManagedBy from ever being set. Was there somewhere else you were thinking of handling this?

Reply:
Yeah I just noticed, I'm trying to figure out why the value set in `ManagedBy`  is available in the new struct after doin Marshal and UnMarshal =(